src/features/map/MapFilterModal.tsx

```diff
@@ -27,11 +32,26 @@ type Props = {
 const MapFilterModal = ({ mapFilter, onChangeMapFilter }: Props) => {
   const { t } = useTranslation()
   const dispatch = useAppDispatch()
+  const colors = useColors()
   const { showMapFilter } = useSelector(
     (state: RootState) => state.hotspotDetails,
   )
+  const networkHotspotEarnings = useSelector(
+    (state: RootState) => state.rewards.networkHotspotEarnings.data,
+  )
   const [selectedFilter, setSelectedFilter] = useState(mapFilter)
 
+  useMount(() => {
+    dispatch(fetchNetworkHotspotEarnings())
+  })
+
+  const avg7dEarnings = useMemo(() => {
+    if (networkHotspotEarnings.length === 0) return
```

Comment:
Probably not a big deal, but it's possible that the earnings job could fail and a day of earnings would be missing

Reply:
This was the same logic as explorer, what would we want to do in the case it fails? It would always just take the last available 7, which should be a pretty close approximate in the case one day fails.